tests/utils/builders/user_builder.py

```diff
@@ -31,12 +36,11 @@ def with_password(self, password: str) -> "UserBuilder":
         return self
 
     def build(self) -> User:
-        user: User = User.objects.create(
-            email=self._email,
-            description=self._description,
-            is_staff=self._is_staff,
-            date_joined=self._date_joined,
-        )
+        user, created = User.objects.get_or_create(email=self._email)
         user.set_password(self._password)
+        user.description = self._description
+        user.is_staff = self._is_staff
+        user.is_superuser = self._is_superuser
+        user.date_joined = self._date_joined
```

Comment:
In a builder, you always want to create the resource. As is only used in tests, you know which users are created, so revert this change.

Reply:
OK.

Reply:
Done.